src/components/ProgressIndicator/ProgressIndicator.stories.tsx

```diff
@@ -0,0 +1,201 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { useEffect, useState } from "react";
+import { withDesign } from "storybook-addon-designs";
+
+import ProgressIndicator, {
+  progressIndicatorSizesArray,
+  progressIndicatorTypesArray,
+} from "./ProgressIndicator";
+import SimpleGrid from "../Grid/SimpleGrid";
+
+const meta: Meta<typeof ProgressIndicator> = {
+  title: "Components/Feedback/ProgressIndicator",
+  component: ProgressIndicator,
+  decorators: [withDesign],
+  argTypes: {
+    darkMode: {
+      control: false,
+    },
+    id: { control: false },
+    indicatorType: {
+      controls: { type: "select" },
+      options: progressIndicatorTypesArray,
+      table: { defaultValue: { summary: "linear" } },
+    },
+    isIndeterminate: {
+      table: { defaultValue: { summary: false } },
+    },
+    showLabel: {
+      table: { defaultValue: { summary: true } },
+    },
+    size: {
+      controls: { type: "select" },
+      options: progressIndicatorSizesArray,
+      table: { defaultValue: { summary: "default" } },
+    },
+    value: {
+      table: { defaultValue: { summary: 0 } },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof ProgressIndicator>;
+
+/**
+ * Main Story for the ProgressIndicator component. This must contains the `args`
+ * and `parameters` properties in this object.
+ */
+export const WithControls: Story = {
+  args: {
+    id: "progressIndicator-id",
+    indicatorType: "linear",
+    isIndeterminate: false,
+    labelText: "Progress",
+    showLabel: true,
+    size: "default",
+    value: 50,
+  },
+  render: (args) => <ProgressIndicator {...args} />,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=37638%3A23842",
+    },
+    jest: ["ProgressIndicator.test.tsx"],
+  },
+};
+
+export const LinearType: Story = {
+  argTypes: {
+    indicatorType: { table: { disable: true } },
+  },
+  render: () => (
+    <ProgressIndicator
+      id="linear"
+      indicatorType="linear"
+      labelText="Linear Progress Type"
+      value={50}
+    />
+  ),
+};
+export const CircularType: Story = {
+  argTypes: {
+    indicatorType: { table: { disable: true } },
+  },
+  render: () => (
+    <ProgressIndicator
+      id="circular"
+      indicatorType="circular"
+      labelText="Linear Progress Type"
```

Comment:
I know that this is the way it is in the current storybook but it might be nice to set the `labelText` to "Circular Progress Type"?

Reply:
Yea, this is wrong. Good catch!